pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -106,7 +112,23 @@ ansiColor('xterm') {
             folioNamespaceCreate(namespaceCreateParams)
           }
         }
-        parallel(branches)
+
+//        Map branches = [failFast: false]
+//        namespacesMap.each { id, namespace ->
```

Comment:
Why do we keep this huge commented block?

Reply:
It will be removed in next PR's